logger_bench_test.go

```diff
@@ -41,7 +41,7 @@ func (u user) MarshalLog(kv zap.KeyValue) error {
 	return nil
 }
 
-var _jane = user{
+var _jane = &user{
```

Comment:
Thanks for explaining this more to me offline. Can we also change the user struct's `MarshalLog` method to have a pointer receiver?